src/api/customerApi.ts

```diff
@@ -12,18 +12,54 @@ export class CustomerService {
         return mergeHalArray<Customer>(embedded);
     }
 
+    //  Acepta CustomerEntity en lugar de Customer
+    async createCustomer(customer: Omit<CustomerEntity, 'uri'>): Promise<Customer> {
+        const resource = await postHal('/customers/register/', customer as any, this.authProvider);
```

Comment:
This endpoint doesn't exist in the backend, it is a RESTful API and to create new instances the approach is to do a POST to /customers. You can use the userApi.ts as an example